README.md

```diff
@@ -75,6 +75,7 @@ Then copy the `.babelrc` in this repo, or the below:
         "@babel/typescript"
     ],
     "plugins": [
+        "@babel/plugin-proposal-numeric-separator",
```

Comment:
I think `plugin-proposal-numeric-separator` should be optional

Reply:
Even better :) I have removed it completely.

Reply:
Looks good.